go/vt/vttablet/tabletserver/vstreamer/vstreamer_test.go

```diff
@@ -398,6 +398,79 @@ func TestMissingTables(t *testing.T) {
 	runCases(t, filter, testcases, startPos, nil)
 }
 
+// TestInternalTables tests streaming of sidecar db tables.
+func TestInternalTables(t *testing.T) {
+	ts := &TestSpec{
+		t: t,
+		ddls: []string{
+			"create table t1(id11 int, id12 int, primary key(id11))",
+			"create table _vt.internal1(id int, primary key(id))",
+			"create table _vt.internal2(id int, primary key(id))",
+		},
+	}
+	ts.Init()
+	defer func() {
+		execStatements(t, []string{
```

Comment:
Any reason not to drop t1, since we created it w/o `IF NOT EXIST`?

Reply:
it is dropped in the framework teardown.